clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -1482,8 +1482,9 @@ The -record_heap parameter requests recording of a pre-determined set
 of functions related to heap allocation.  The -record_heap_value
 paramter controls the contents of this set.
 
-The tracer also supports recording system call argument and success
-values via the option -record_syscall, which functions similarly to
+The tracer also supports recording system call argument and return
+values (along with whether the call failued) via the option -record_syscall, which
+functions similarly to
```

Comment:
nit: s/functions/works/
since "function" is already used in this sentence with a different meaning.